### DISCUSSION TITLE
How to add icons with toast?

### DISCUSSION THREAD

**Discussion by rishi-raj-jain** [zeit-ui/react#446] [Ideas]
Loving the library, while was stuck with adding icons to say left/right corner of the toast. Is that possible?

**Comment by unix** [comment]
The current design of `Toast` is closed and no elements can be added for now. There is a plan to refactor `Toast` in issues, but it cannot be completed in the short term.

**Reply by rishi-raj-jain** [reply]
No prob, thanks!
Great library, literally.
